models/keyword.go

```diff
@@ -0,0 +1,59 @@
+package models
+
+import (
+	"errors"
+
+	"github.com/gutakk/go-google-scraper/db"
+	errorHandler "github.com/gutakk/go-google-scraper/helpers/error_handler"
+	"gorm.io/gorm"
+)
+
+const (
+	fileFormatError  = "File must be CSV format"
+	fileLengthError  = "CSV file must contain between 1 to 1000 keywords"
+	invalidDataError = "Invalid data"
+)
+
+type Keyword struct {
+	gorm.Model
+	Keyword string `gorm:"notNull;index"`
+}
+
+func ValidateFileType(fileType string) error {
+	if fileType != "text/csv" {
+		return errors.New(fileFormatError)
+	}
+	return nil
+}
+
+func ValidateCSVLength(row int) error {
+	if row <= 0 || row > 1000 {
+		return errors.New(fileLengthError)
+	}
+	return nil
+}
+
+func SaveKeywords(record [][]string) ([]Keyword, error) {
+	var keywords = []Keyword{}
+
+	// Check if record more than 0
+	if len(record) <= 0 {
```

Comment:
Can this be really negative?

Reply:
No it doesn't. Fixed in 56662ee0b4df157d4d83680122d71b54be3a45aa